instrument/ARGUS_Definition_2021-05-18.xml

```diff
@@ -0,0 +1,482 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- For help on the notation used to specify an Instrument Definition File 
+     see http://www.mantidproject.org/IDF -->
+<instrument xmlns="http://www.mantidproject.org/IDF/1.0" 
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 http://schema.mantidproject.org/IDF/1.0/IDFSchema.xsd"
+ name="ARGUS" valid-from   ="1995-02-28 23:59:59"
```

Comment:
Can I clarify how to interpret the valid-from date here with the one in `ARGUS_Definition.xml`? They overlap so it seems ambiguous what should be picked.

Reply:
Would updating the old definition file's valid to date to 2021-05-18 and changing the new file valid from date to 2021-05-18 suffice? Since this file was supplied by James I didn't want to break anything

Reply:
I have asked for some clarification on what to do with the old file

Reply:
After asking James the old definition file can be replaced as the new file should be valid for all ARGUS data to date